docs/guides/customize-user-lockout.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Customize max login attempts and duration of user lockout
+description: 
+---
+
+# Customize the max attempts before getting locked out and duration of lock out
+
+<Callout type="info">
+This feature is only available for user accounts that use [passwords](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) and [backup codes](https://dashboard.clerk.com/last-active?path=user-authentication/multi-factor).
```

Comment:
"Or" backup codes (or both)

Reply:
Since we don't conditionally disable User Lockout if neither password or backup code is available, I'd rephrase this part to:

>This feature is applicable to user accounts that use passwords or backup codes.